packages/shared/components/OnboardingLayout.svelte

```diff
@@ -12,6 +14,11 @@
     // TODO: Separate mobile
     let mobileTopContentHeight,
         leftpaneContentHeight = 0
+
+    let isMacOs = false
+    onMount(async () => {
+        isMacOs = (await Platform.getOS()) === 'darwin'
+    })
```

Comment:
We do this check in multiple places, lets create a task to refactor into a store we can use across the app